tardis/adapters/sites/moab.py

```diff
@@ -120,9 +121,10 @@ def __init__(self, machine_type: str, site_name: str):
     async def deploy_resource(
         self, resource_attributes: AttributeDict
     ) -> AttributeDict:
+        msub_cmdline_option_string = self.msub_cmdline_options()
         request_command = (
-            f"msub -j oe -m p -l "
-            f"walltime={self.machine_type_configuration.Walltime},"
+            f"msub -j oe -m p {msub_cmdline_option_string}"
```

Comment:
Would it be possible to include all options in the `AttributeDict`s passed to `submit_cmdline_options` call like in 
https://github.com/MatterMiners/tardis/blob/1139be80d885305ef8a62fca7ca49d8e15ca334e/tardis/adapters/sites/slurm.py#L185-L198 ?